DevCycle/Models/PlatformDetails.swift

```diff
@@ -56,3 +56,12 @@ func getMacOSModelIdentifier() -> String {
     }
 }
 #endif
+
+#if os(iOS) || os(tvOS) || os(watchOS)
+func getDeviceModel() -> String {
+    if let simulatorModelIdentifier = ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"] { return simulatorModelIdentifier }
```

Comment:
Interesting, what does this look like? Not sure we should set the simulator device model, might be better to have it be just the generic x86 simulator model name.

Reply:
this just returns the data in the format like a real device (aka `iPhone 15,7`) instead of having the device marked as a simulator

Reply:
yea I think we actually want it marked as a simulator. Often with TL we see really odd behaviour from simulators in our data (usually due to automated testing), so I'd prefer to know that its a simulator.

Reply:
I checked without this, and it simply returns it as `arm64` or the architecture of the mac running the simulator.. instead of that, I'll update it to explicitly mark them as simulators and still return the right model information